includes/wc-deprecated-functions.php

```diff
@@ -867,3 +867,18 @@ function _wc_save_product_price( $product_id, $regular_price, $sale_price = '',
 		update_post_meta( $product_id, '_sale_price_dates_to', '' );
 	}
 }
+
+/**
+ * Get a log file path.
+ *
+ * @since 2.2
+ * @deprecated 2.7.0
+ *
+ * @param string $handle name.
+ * @return string the log file path.
+ */
+function wc_get_log_file_path( $handle ) {
```

Comment:
I'm not a big fan of deprecated a function in favor of a static method.
The use of this function is more easy than remember the class name and method.
@mikejolley thoughts?